code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -462,12 +654,22 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	icon_state = "deck_syndicate_full"
 	var/used = FALSE
 	var/theme = "magic"
-	var/mob_name = "Guardian Spirit"
+	var/mob_name = "guardian"
 	var/use_message = "<span class='holoparasite'>You shuffle the deck...</span>"
 	var/used_message = "<span class='holoparasite'>All the cards seem to be blank now.</span>"
 	var/failure_message = "<span class='holoparasitebold'>..And draw a card! It's...blank? Maybe you should try again later.</span>"
 	var/ling_failure = "<span class='holoparasitebold'>The deck refuses to respond to a souless creature such as you.</span>"
-	var/list/possible_guardians = list("Assassin", "Chaos", "Charger", "Explosive", "Lightning", "Protector", "Ranged", "Standard", "Support")
+
+	var/list/chosen_abilities = null
```

Comment:
Why are these variables even here ? They are only used in creation proc as far as i can tell ?